services/codeclimate/codeclimate.tester.js

```diff
@@ -110,7 +110,7 @@ t.create('malformed response for outer user repos query')
 t.create('malformed response for inner specific repo query')
   .get('/maintainability/angular/angular.js.json')
   .intercept(nock => nock('https://api.codeclimate.com', {allowUnmocked: true})
-    .get('/v1/repos/526933117e00a40567008444/snapshots/5ab4427859c16c0001003cad')
+    .get(/\/v1\/repos\/526933117e00a40567008444\/snapshots\/[a-z0-9]+/)
```

Comment:
Would `526933117e00a40567008444` possibly change in the future aswell?
(on mobile right now so can't check)

Reply:
Good shout. :+1: I've broadened it so both params can be `[a-z0-9]+`